src/Microsoft.AspNetCore.Server.Kestrel/Filter/Internal/FilteredStreamAdapter.cs

```diff
@@ -4,11 +4,11 @@
 using System;
 using System.IO;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Server.Kestrel.Http;
-using Microsoft.AspNetCore.Server.Kestrel.Infrastructure;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Filter
+namespace Microsoft.AspNetCore.Server.Kestrel.Filter.Internal
```

Comment:
Why `Internal.Http` and `Internal.Infrastructure` but `Filter.Internal`? Is it just because there is already public stuff in the Filter namespace, but not the HTTP and Infrastructure namespace?